sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -1371,21 +1373,35 @@ trait ParquetPartitionReaderBase extends Logging with Arm with ScanWithMetrics
       clippedReadFields.map { f =>
         if (useFieldId && ParquetSchemaClipShims.hasFieldId(f)) {
           // find the parquet column name
-          fieldIdToNameMap(ParquetSchemaClipShims.getFieldId(f))
+          (fieldIdToNameMap(ParquetSchemaClipShims.getFieldId(f)), f)
         } else {
-          m.get(f.name).getOrElse(f.name)
+          (m.get(f.name).getOrElse(f.name), f)
         }
       }
     } else {
       clippedReadFields.map { f =>
         if (useFieldId && ParquetSchemaClipShims.hasFieldId(f)) {
-          fieldIdToNameMap(ParquetSchemaClipShims.getFieldId(f))
+          (fieldIdToNameMap(ParquetSchemaClipShims.getFieldId(f)), f)
         } else {
-          f.name
+          (f.name, f)
         }
       }
     }
   }
+
+  val sparkToParquetSchema = new SparkToParquetSchemaConverter()
+  def getParquetOptions(clippedSchema: MessageType, useFieldId: Boolean): ParquetOptions = {
+    val includeColumns = toCudfColumnNamesAndDataTypes(readDataSchema, clippedSchema,
+      isSchemaCaseSensitive, useFieldId)
+    val builder = ParquetOptions.builder().withTimeUnit(DType.TIMESTAMP_MICROSECONDS)
+    includeColumns.foreach { t =>
+      builder.includeColumn(t._1,
+        t._2.dataType == BinaryType &&
+            sparkToParquetSchema.convertField(t._2).asPrimitiveType().getPrimitiveTypeName
+                == PrimitiveTypeName.BINARY)
```

Comment:
Build is failing here:
```
Error: ] /home/runner/work/spark-rapids/spark-rapids/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala:1399: type mismatch;
 found   : Boolean
 required: String
Error:  one error found
```

Reply:
Where? I just built it again locally and it's passing. 

Do you have the cudf changes to ParquetOptions?

Reply:
It's failing in CI (see failed runs below), although it looks like that was run against 22.10?  It was trying to download spark-rapids-jni-22.10-SNAPSHOT.

It looks like the normal CI is running against 22.08 correctly, so I'll try to manually re-kick the ones that failed here.